docs/installation/requirements.rst

```diff
@@ -10,7 +10,7 @@ Here is the list of requirements:
 -   :code:`dnspython`
 -   :code:`domain2idna`
 -   :code:`inflection`
--   :code:`PyMySQL`like - for example - Travis CI
+-   :code:`PyMySQL` like - for example - Travis CI
```

Comment:
Uhh can you please delete the `like - for example - Travis CI`. I can't remember how it ended there 🤔

Reply:
Sure thing, I was kind a wondering the same thing.